config/config.php

```diff
@@ -74,6 +74,9 @@
 $CDASH_BERNARD_DRIVER = false;
 $CDASH_BERNARD_CONSUMERS_WHITELIST = false;
 
+// EXPERIMENTAL: Whether or not to use Bernard for submitting coverage jobs
+$CDASH_BERNARD_COVERAGE_SUBMISSION = false;
```

Comment:
Correct. For now it's `$CDASH_BERNARD_SUBMISSION` corresponds to non-coverage submissions and `$CDASH_BERNARD_COVERAGE_SUBMISSION` corresponds to coverage-only submissions.